pkgs/development/compilers/ponyc/default.nix

```diff
@@ -24,32 +24,31 @@ stdenv.mkDerivation (rec {
   };
 
   ponygbenchmark = fetchurl {
```

Comment:
This should use fetchFromGitHub.

Reply:
```
these derivations will be built:
  /nix/store/92c72baifbp9s3mmqnbpx2qrpadasgwg-ponyc-0.41.1.drv
building '/nix/store/92c72baifbp9s3mmqnbpx2qrpadasgwg-ponyc-0.41.1.drv'...
unpacking sources
unpacking source archive /nix/store/m13695q8qmhfkhxk901sk5lb9vn6g1nb-source
source root is source
tar (child): /nix/store/irnffa1r8vvvgqp498gz19z1km8vz0gf-source: Cannot read: Is a directory
tar (child): At beginning of tape, quitting now
tar (child): Error is not recoverable: exiting now

gzip: stdin: unexpected end of file
tar: Child returned status 2
tar: Error is not recoverable: exiting now
builder for '/nix/store/92c72baifbp9s3mmqnbpx2qrpadasgwg-ponyc-0.41.1.drv' failed with exit code 2
error: build of '/nix/store/92c72baifbp9s3mmqnbpx2qrpadasgwg-ponyc-0.41.1.drv' failed
```

Well, looks like fetchFromGitHub results in a directory in /nix/store being created instead of a file.  It has to be copied into my build directory... Doing a deep copy of one directory into another doesn't feel right to me.

Is this your intent?

Reply:
Conversations on Matrix - updating in a moment

Reply:
Pushed